backend/app/auth/handlers.go

```diff
@@ -1,13 +1,120 @@
 package auth
 
 import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"github.com/gin-gonic/contrib/sessions"
 	"github.com/gin-gonic/gin"
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+	githuboauth "golang.org/x/oauth2/github"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"time"
 )
+// auth configurations
+var oauthConf = &oauth2.Config{
+	ClientID:     os.Getenv("GITHUB_CLIENT_ID"),
+	ClientSecret: os.Getenv("GITHUB_CLIENT_SECRET"),
+	Scopes:       []string{"user"},
+	Endpoint:     githuboauth.Endpoint,
+}
+var secretKey = os.Getenv("SECRET_KEY_BASE")
```

Comment:
Wouldn't it be better to fetch this `secretKey` from the env directly in this file ` backend/app/auth/routers.go` on line 13 ?

Reply:
Updated !